app/service/data_svc.py

```diff
@@ -198,11 +202,14 @@ def __init__(self):
     async def _load_abilities(self, plugin):
         for filename in glob.iglob('%s/abilities/**/*.yml' % plugin.data_dir, recursive=True):
             for entries in self.strip_yml(filename):
+                # ab is an ability dict
```

Comment:
They are trying to keep comments out of the code.  They would prefer that the names of the objects in the code provide descriptive attributes instead.